common/data_refinery_common/models/accession.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+from django.db import models
+from django.utils import timezone
+
+
+class AccessionBacklogEntry(models.Model):
```

Comment:
Lets update the classname, file, and db_table to a variation of GatheredAccession for now.